docs/asciidoc/modules/ROOT/pages/database-integration/vectordb/chroma.adoc

```diff
@@ -7,6 +7,7 @@ note that the list and the signature procedures are consistent with the others,
 [opts=header, cols="1, 3"]
 |===
 | name | description
+| apoc.vectordb.chroma.info(hostOrKey, collection, $config) | Get information about specified existing collection
```

Comment:
Add information about the output when the collection does not exists (return null? throws error?)

Reply:
Added, i.e. `throws an error if it does not exist`